src/aks-preview/azcli_aks_live_test/configs/ext_matrix_default.json

```diff
@@ -26,10 +26,6 @@
         "overlay migration, missing toggle": [
             "test_aks_azure_cni_overlay_migration"
         ],
-        "service mesh, missing feature registration": [
-            "test_aks_azure_service_mesh_enable_disable",
```

Comment:
Is the operation not guarded by AFEC feature anymore? I see there's enable step in the test case and it seems the cmd still does not support passing a custom feature header. Test sub used by CLI runner has no feature registered except AKSHTTPCustomFeature, if the test case could not bypass feature validation on rp side, the test would always fail, and I would recommend keeping the case excluded

> update_cmd = 'aks mesh enable --resource-group={resource_group} --name={name}'

Reply:
this is correct, feature is no longer behind AFEC feature flag.